info.febvre.Komikku.json

```
@@ -3,10 +3,14 @@
     "runtime" : "org.gnome.Platform",
     "runtime-version" : "master",
     "sdk" : "org.gnome.Sdk",
+    "sdk-version" : "master",
```

```suggestion
    "runtime-version" : "42",
    "sdk" : "org.gnome.Sdk",
```

There is no `master` version of the GNOME SDK.

---

This is what I have deduced.

Unfortunately, the app needs GTK 4.7.1, libadwaita 1.2.beta and webkit2gtk-5. Runtime 42 is not suitable.

Thanks for the confirmation.